### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by networkfusion**
## Description

Split documentation into chunks that users can understand.
Improve them to be more idiot proof!

## Motivation and Context

Allows an easier way to point people towards specific documentation.
#127 
#76 

## Types of changes

- [ ] Improvement (non-breaking change that adds a new feature)
- [ ] Bug fix (fixes an issue)
- [ ] Breaking change (breaking change)
- [x] Documentation Improvement
- [ ] Config and build (change in the configuration and build system, has no impact on code or features)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

Signed-off-by: GITHUB_USER <GITHUB_USER_EMAIL>

### CODE REVIEW

File: docs/00_getting_started_sd.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+ - **SNES**: [sodium64](https://github.com/Hydr8gon/sodium64/releases) by *Hydr8gon* - `sodium64.z64`
+ - **Game Boy** / **GB Color**: [gb64](https://lambertjamesd.github.io/gb64/romwrapper/romwrapper.html) by *lambertjamesd* - `gb.v64` / `gbc.v64` ("Download Emulator" button)
+
+
+### 64DD disk support
+For the ability to load and run 64DD disk images, you need to place required 64DD IPL dumps in the `/menu/64ddipl` folder on the SD card.
+For more details follow [this guide on the 64dd.org website](https://64dd.org/tutorial_sc64.html).
+
+
+### So what would the layout of the SD Card look like?
```
Comment (networkfusion) [inline docs/00_getting_started_sd.md:32]:
```suggestion
#### So what would the layout of the SD Card look like?
```

**Comment by Polprzewodnikowy** [comment]
how does this work with doxygen generated page? will these subpages be available at https://menu.summercart64.dev ?

**Comment by Polprzewodnikowy** [comment]
also I think we need to prepare manually written page for the `menu.` subdomain and keep doxygen stuff on other URL

**Comment by networkfusion** [comment]
> how does this work with doxygen generated page? will these subpages be available at https://menu.summercart64.dev ?

I dont know... but doxygen no longer reports an error about it (after the latest commit).

**Comment by Polprzewodnikowy** [comment]
can we merge this and test how doxygen behaves? you could create new PR for further improvements

**Comment by networkfusion** [comment]
> can we merge this and test how doxygen behaves? you could create new PR for further improvements

Yes, after that last one, I think we are ready! ( I am playing around the edges now)!

**Comment by Polprzewodnikowy** [comment]
cool, subpages are working

**Pull request merged**